operating/manage-security-policies.adoc

```diff
@@ -0,0 +1,52 @@
+[id="manage-security-policies"]
+= Managing security policies
+include::modules/common-attributes.adoc[]
+:context: manage-security-policies
+
+toc::[]
+
+[role="_abstract"]
+{product-title} allows you to use out-of-the-box security policies and define custom multi-factor policies for your container environment.
+Configuring these policies enables you to automatically prevent high-risk service deployments in your environment and respond to runtime security incidents.
+
+include::modules/use-default-security-policies.adoc[leveloffset=+1]
+
+include::modules/modify-existing-security-policies.adoc[leveloffset=+1]
+
+include::modules/create-policy-categories.adoc[leveloffset=+1]
+
+[id="create-custom-policies"]
+== Creating custom policies
+
+In addition to using the default policies, you can also create custom policies in {product-title}.
+
+To build a new policy you can clone an existing policy, or create a new one from scratch.
```

Comment:
Sorry, should have been more specific. The second comma wasn't necessary. There should have still been one after "To build a new policy":

```
To build a new policy, you can clone an existing policy or create a new one from scratch.
```